### PR TITLE
[Provider: Titus ] Use instaces(Min/Max/Desired) and add a resize operation

### PR DESCRIPTION
**Pull request by tomaslin**

### CODE REVIEW

**Comment by tomaslin** [comment]
@dzapata @cfieber PTAL

File: clouddriver-titus/src/main/groovy/com/netflix/spinnaker/clouddriver/titus/client/RegionScopedTitusClient.java [hunk 1]
```diff
@@ -156,6 +156,14 @@ public Task getTask(String taskId) {
     }
 
     @Override
+    public void resizeJob(ResizeJobRequest resizeJobRequest){
+        if(resizeJobRequest.getUser() == null){
```
Comment (dzapata) [inline clouddriver-titus/src/main/groovy/com/netflix/spinnaker/clouddriver/titus/client/RegionScopedTitusClient.java]: In which instances would this be null?

**Comment by dzapata** [comment]
LGTM

File: clouddriver-titus/src/main/groovy/com/netflix/spinnaker/clouddriver/titus/client/RegionScopedTitusClient.java (hunk 1 above)
Comment (tomaslin) [inline clouddriver-titus/src/main/groovy/com/netflix/spinnaker/clouddriver/titus/client/RegionScopedTitusClient.java]: in the cases where authentication is disabled. Could happen when Titus is OSS'd

Comment (tomaslin) [inline clouddriver-titus/src/main/groovy/com/netflix/spinnaker/clouddriver/titus/client/RegionScopedTitusClient.java]: also, duplicating the functionality in submitjobs here

**Pull request merged**